command/helpers/helpers.go

```diff
@@ -124,13 +126,19 @@ func newDecodeConfigEntry(raw map[string]interface{}) (api.ConfigEntry, error) {
 	}
 
 	for _, k := range md.Unused {
-		switch k {
-		case "kind", "Kind":
+		switch {
+		case strings.ToLower(k) == "kind":
```

Comment:
I changed this up so that the error messaging would be the same whether doing a `consul config write` as we get from the api